changelog.json

```diff
@@ -1,6 +1,14 @@
 {
   "repository": "newrelic/node-newrelic",
   "entries": [
+    {
+      "version": "10.6.0",
+      "changes": {
+        "security": [],
```

Comment:
This is the first release where we haven't had any front-matter related changes fyi

Reply:
does it matter?

Reply:
it shouldn't, was just a fyi in case it does (which would be a bug we'd have to fix)

Reply:
10.4.0 had nothing 

```
    {
      "version": "10.4.0",
      "changes": {
        "security": [],
        "bugfixes": [],
        "features": []
      }
    },


```

Reply:
oh good, sorry for the false alarm then :homerbush: